Diff/Diff.sublime-syntax

```diff
@@ -25,18 +25,18 @@ first_line_match: |-
 
 contexts:
   main:
-    - match: ^(?={{git_first_line}})
+    - match: '{{bol}}(?={{git_first_line}})'
       embed: Packages/Git Formats/Git Diff.sublime-syntax#email-first-line
       escape: (?!)
     # Hack for unit tests
     - include: incomplete-line
-    - match: ^
+    - match: '{{bol}}'
```

Comment:
Not sure this line need a change. It may even cause any other diff rule on line 1 not being matched, in case `bol: ^\s*` is specified.